ga4gh/datamodel/genotype_phenotype.py

```diff
@@ -0,0 +1,338 @@
+"""
+Module responsible for translating g2p data into GA4GH native
+objects.
+"""
+
+import ga4gh.protocol as protocol
+import ga4gh.exceptions as exceptions
+from urlparse import urlparse, urlunsplit
```

Comment:
Elsewhere we use 
```
import urlparse
```
and call ``urlparse.urlparse()`` and ``urlparse.urlunsplit()`` in the code. This helps us see where functions are being imported from without having to hunt back up to the header (I agree this is pretty marginal benefit for something like urlparse.urlparse(), but consistency is nice...)